AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java

```diff
@@ -227,6 +227,12 @@ private void initAllContentViews() {
         Uri url = Uri.parse(getResources().getString(R.string.register_url));
         signUpButton.setOnClickListener(v -> openUrl(url));
 
+        //Add new button to link to instructions on how to find AnkiWeb email
```

Comment:
'new' is time relative, this comment will hopefully be here for 10 years :)

```suggestion
        // Add button to link to instructions on how to find AnkiWeb email
```

Reply:
Removed "new" in comment